template/cordova/lib/prepare.js

```diff
@@ -353,6 +391,13 @@ function mapImageResources(images, imagesDir) {
                 events.emit('warn', 'No images found for target: ' + img.target);
             } else {
                 candidates.forEach(function(mrtImage) {
+                    if (img.target === SPLASH_SCREEN_DESKTOP_TARGET_NAME &&
+                        mrtSplashScreenToTargetProject(mrtImage) === TARGET_PROJECT_10 &&
+                        exceedsSizeLimit(mrtImage.path)) {
```

Comment:
This does not result in a build failure and the issue could only be seen in Visual Studio manifest designer so I believe the warning is useful here.
Do you mean we should fail the build rather than showing this warning?
It currently just skips oversized images.